code/modules/mob/living/carbon/human/species.dm

```diff
@@ -58,6 +58,13 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/sound/attack_sound = 'sound/weapons/punch1.ogg'
 	var/sound/miss_sound = 'sound/weapons/punchmiss.ogg'
 
+	var/sound/Hfemale_scream_sound = 'sound/effects/mob_effects/goonstation/female_scream.ogg'
```

Comment:
This is like this because of vox sounds in sunset/code/modules/mob/living/carbon/human/species the H stands for "Human"

Reply:
I may remove the scream sounds as i don't need them with tg scream code

Reply:
I decided to remove these var's from the species.dm and use the straight paths